python-stdlib/logging/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .logging import Logger
+from .logging import Handler
+from .logging import Formatter
+from .logging import FileHandler
+from .logging import StreamHandler
+from .logging import DEBUG, INFO, WARNING, ERROR, CRITICAL
+from .logging import debug, info, warning, error, critical
+from .logging import exception
+from .logging import shutdown
+from .logging import getLogger
+from .logging import basicConfig
+
+import sys
+
+if hasattr(sys, "atexit"):
+    sys.atexit(shutdown)
```

Comment:
This file is not really needed anymore, it is simpler just to have `logging.py` as a stand-alone one-file library (like it was previously).

Reply:
I think it was cleaner to keep `__init_.py` especially if it imports handlers in the future, but okay.